Adds Note for new users to follow dev-guide as a possible option to i… (#7995)
sunpy/sunpy: docs/tutorial/installation.rst

This is the first chapter in the sunpy tutorial, and by the end of it you should have a working installation of Python and ``sunpy``.
For further information and alternative methods for installing ``sunpy`` beyond the recommended approach outlined below, refer to :ref:`sunpy-topic-guide-installing`.

> Note: We encourage all new and existing users to contribute code to this community project. Please consider following the developer installation instructions

Reviewer: ```suggestion
.. note::
   We encourage all new and existing users to contribute code to this community project. 
   Please consider following the :ref:`developer installation instructions <_newcomers>`.
```

> instead which are only slightly more difficult, refer to :ref:`newcomers`. These instructions install sunpy in a way that does not enable editing sunpy code. It is, of course, possible

Reviewer: ```suggestion
   It is slightly more difficult and will make it easier to contribute code in the future.
```

Reviewer: Thanks @tanishy7777! I put this inside a note sphinx block (or at least I tried assuming I did the spacing correctly). See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html.

This is the first chapter in the sunpy tutorial, and by the end of it you should have a working installation of Python and ``sunpy``.
For further information and alternative methods for installing ``sunpy`` beyond the recommended approach outlined below, refer to :ref:`sunpy-topic-guide-installing`.

> .. note::
>    We encourage all new and existing users to contribute code to this community project. 
>    Please consider following the :ref:`developer installation instructions <_newcomers>`.
>    It is slightly more difficult and will make it easier to contribute code in the future.
> to do this later!
> 

Reviewer 2: ```suggestion
.. note::

   We encourage all new and existing users to contribute code to this community project. 
   Please consider following the :ref:`developer installation instructions <newcomers>`.
   It is slightly more difficult but will make it easier to contribute code in the future.

```

This link will never work as the newcomers reference only exists in the latest version and never the released version of the documentation.

I also do not see the point of this.
